dotnet/src/InternalUtilities/planning/PlannerInstrumentation.cs

```diff
@@ -160,11 +171,22 @@ internal static partial class PlannerInstrumentation
         Message = "Plan executed successfully.")]
     static partial void LogInvokePlanSuccess(this ILogger logger);
 
-    [LoggerMessage(
-        EventId = 8,
-        Level = LogLevel.Trace, // Sensitive data, logging as trace, disabled by default
-        Message = "Plan result: {PlanResult}")]
-    static partial void LogPlanResult(this ILogger logger, object planResult);
+    private static readonly Action<ILogger, string, Exception?> s_logPlanResult =
+        LoggerMessage.Define<string>(
+            logLevel: LogLevel.Trace,   // Sensitive data, logging as trace, disabled by default
+            eventId: 8,
```

Comment:
CAn we have an enumerable or a description of what `8` really is?

Reply:
These numbers don't carry any meaning at the moment. I think we need to audit the entire SK project (all places where logs are generated) to determine the event code for each event if we want to identify the events. For now, I have change all of them to 0, which is the default if you are just using `ILogger.LogXXX`. And I have also suppressed the warning.